.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: CI
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [ '2.1.9', '2.2.10', '2.3.7' ]
```

Comment:
This keeps the ancient EOL versions, which is a valid 1st step, for "parity" I guess.

Reply:
Yes, that's the idea here, just replace the tool and then, in future coming PRs, support newer Ruby versions and drop EOL.

Reply:
(This happened with emojis: GitHub's emoji picker _moves_ under my pointer, and made me pick the _wrong react_. This is the emoji picker nightmare scenario.)